packages/cli/src/constructs/check-group.ts

```diff
@@ -63,7 +63,7 @@ export interface CheckGroupProps {
   concurrency?: number
   environmentVariables?: Array<EnvironmentVariable>
   /**
-   * List of alert channel subscriptions.
+   * List of alert channels this check should subscribe to.
```

Comment:
This is a check group rather than a check. Also I'm not sure if it makes sense to say that the group is subscribed to the alert channel, it's the other way around?

```suggestion
   * List of alert channels to be alerted when checks in this group fail or recover.
```